```
./test-all.sh: 115: ./test-unicorn-mode.sh: Syntax error: "}" unexpected (expecting "fi")
make: *** [GNUmakefile:373: tests] Error 2
Error: Process completed with exit code 2.
```

--- a/test/test-unicorn-mode.sh
+++ b/test/test-unicorn-mode.sh
@@ -86,7 +86,7 @@ test -d "$DIR" && {
           ../afl-fuzz -m ${MEM_LIMIT} -V35 -U -i in -o out -d -s 0 -- "$PY" ../unicorn_mode/samples/compcov_x64/compcov_test_harness.py @@ >>errors 2>&1
           unset UNICORN_AFL_CMPCOV
         } >>errors 2>&1
-        {test -n "$( ls out/default/queue/id:000001* 2>/dev/null )" ||  test -n "$(ls out/default/crashes/id:* 2>/dev/null )"; } && {
+        { test -n "$( ls out/default/queue/id:000001* 2>/dev/null )" || test -n "$(ls out/default/crashes/id:* 2>/dev/null )"; } && {
           $ECHO "$GREEN[+] afl-fuzz is working correctly with unicorn_mode compcov"
         } || {
           echo CUT------------------------------------------------------------------CUT

